torch/quantization/quantization_mappings.py

```diff
@@ -212,6 +218,11 @@ def get_quantized_operator(float_op: Union[Callable, str]) -> Callable:
         'Operator {} does not have corresponding quantized op'.format(str(float_op))
     return quantized_op
 
+# Set of `nn` modules that have custom module implemented
+# This is used for warning the user that they should use custom modules
+def get_supported_custom_modules() -> Set[Callable]:
```

Comment:
I thought the custom modules API was for things users want to do which are not in the PyTorch repo.  Can we just make our APIs handle these modules automatically without requiring the user to know about the custom APIs?  If we need to special case MHA and LSTMs inside the quantization passes to make it invisible to the user, I think that would be good.

Reply:
I think so, cc @z-a-f maybe we can support this without talking about custom module apis (we can have custom module apis to include them by default)

Reply:
can we just always swap MHA with qMHA when default quantization APIs are called, if that is the right way to quantize MHA? (saving the "should we have qMHA" debate for another day)

Reply:
> can we just always swap MHA with qMHA when default quantization APIs are called, if that is the right way to quantize MHA? (saving the "should we have qMHA" debate for another day)

We could do that!